lib.ts

```diff
@@ -124,78 +138,95 @@ export type JobOptions = {
  */
 export class Job {
   /**
-   * Do not used. The _uuid property is unstable and should be considered an internal implementation detail.
-   * @internal
+   * Do not use. The _uuid property is unstable and should be considered an internal implementation detail.
    */
-  readonly _uuid: string;
+  protected readonly _uuid = crypto.randomUUID();
```

Comment:
We actually need to grab this value elsewhere so it cant be `protected` at the moment